ci/playbooks/files/important-logs.html

```diff
@@ -31,16 +31,16 @@
 </style>
 <h3>Logs of interest</h3>
 <ul>
-  <li><a href="./ci-framework-data/logs">All Framework logs</a></li>
-  <li><a href="./ci-framework-data/logs/crc">CRC dedicated logs</a></li>
-  <li><a href="./ci-framework-data/logs/edpm">EDPM logs and data</a></li>
+  <li><a href="./ci-framework-data/logs/">All Framework logs</a> ( shows ci_make logs as well as crc and edpm subdirectories)</li>
+  <li><a href="./ci-framework-data/logs/crc/">CRC dedicated logs</a> (hard copy of the CRC logs, directly from the VM)</li>
+  <li><a href="./ci-framework-data/logs/edpm/">EDPM logs and data</a> (logs about the external dataplane)</li>
```

Comment:
currently this also contains logs about control plane(eg pods detail and logs). Probably we can update this to `logs about k8s resources and external dataplane` ?

Reply:
sounds good, will update shortly!